ecosystem/wallet-apps/overview.mdx

```diff
@@ -2,6 +2,53 @@
+
+  **Wallet contracts** are separate on-chain smart contracts.
+</Aside>
+
+## Wallet types
+
+Wallets are classified by how they manage private keys, store them, and their intended usage.
+
+### Custodial and non-custodial
+
+Wallets are defined by _how private keys are managed_.
+
+- **Custodial wallets** store private keys with a third party, such as an exchange or service provider. The custodian manages security and may provide account recovery. However, users must trust the custodian with full access to their assets.
+- **Non-custodial or self-custody wallets** store private keys only with the user. Keys are protected with a **seed phrase** — a sequence of 24 words generated at wallet creation. This gives the user _complete ownership_ and _control over their data_. However, full responsibility also applies: losing the seed phrase means permanent loss of access.
+
+### Hardware and software
+
+The storage of private keys defines wallet types — _online (hot) or offline (cold)_.
+
+- **Hardware (cold) wallets** are physical devices that keep private keys offline, isolated from internet exposure. When authorizing a transaction, the device signs it internally, ensuring that private keys remain on the device.
+- **Software (hot) wallets** run on devices such as smartphones or computers. Private keys are stored in the device’s secure storage, making them readily available for transactions but also exposed to online risks. Software wallets include **mobile**, **web**, and **desktop** applications.
+
+### Developer and user
+
+Wallets differ by their intended use.
+
+- **User wallets** are designed for everyday operations, including storing, sending, and receiving Toncoin or tokens, as well as interacting with applications. Their design emphasizes usability and accessibility.
+- **Developer wallets** are designed for testing and development purposes. They are typically used with Testnet, SDKs, or command-line tools to deploy contracts, generate keys, and send transactions in controlled environments.
+
+<Aside
+  type="caution"
+>
+  Developer wallets, such as TonDevWallet, are intended for Testnet only. Do **not** use them on Mainnet, as this may result in irreversible loss of funds.
+</Aside>
+
+## Wallet apps
+
+| Wallet app                                    |                                  Description                                 | Creator     |
+| --------------------------------------------- | :--------------------------------------------------------------------------: | ----------- |
+| [Tonkeeper](/ecosystem/wallet-apps/tonkeeper) |        Self‑custodial wallet supports jettons, NFTs, and TON Connect.        | Open-source |
+| [TonDevWallet](/ecosystem/wallet-apps/dev)    |      Self‑custodial desktop wallet designed for development and testing.     | Open-source |
+| [wallet.ton.org](/ecosystem/wallet-apps/web)  | Self-custodial wallet supports jettons, NFTs, TON DNS, TON Sites, TON Proxy. | TON Core    |
```

Comment:
Do not compare them, because they solve different problems. Also need to clarify creators or remove them.

Reply:
Tonkeeper - mobile app
TonDevWallet - developer app
wallet.ton.org - web app